tyr-runner/src/test/java/org/jboss/tyr/command/CommandTest.java

```diff
@@ -34,23 +34,23 @@
     public static final String PR_AUTHOR = "prUser";
     public static final String COMMENT_USER = "commentUser";
 
-    static TestCI testCI;
     static File userListFile;
     private static File adminListFile;
 
     @Inject
     WhitelistProcessing whitelistProcessing;
 
+    @Inject
+    TestCI testCI;
+
     @BeforeAll
     public static void beforeAll() {
-        userListFile = new File(System.getProperty(Utils.TYR_CONFIG_DIR), Utils.USERLIST_FILE_NAME);
-        adminListFile = new File(System.getProperty(Utils.TYR_CONFIG_DIR), Utils.ADMINLIST_FILE_NAME);
+        userListFile = new File(TestUtils.TARGET_DIR, Utils.USERLIST_FILE_NAME);
```

Comment:
I am missing probably where this is picked up? The new name of the property is tyr.users.dir right? If not set the current directory is used which can be /target but it would be better to set the property to not rely on it.

Reply:
It is picked up from application.properties file [https://github.com/jboss/tyr/blob/6bd501d52c268c11d2e7022902f99cf15e02ebb6/tyr-runner/src/test/resources/application.properties#L2](url)